.gitlab/ci/global.yml

```diff
@@ -158,7 +158,7 @@
 .default-job-settings:
   interruptible: true
   extends:
-    - .default-cache
+    # - .default-cache
```

Comment:
intentional?

Reply:
was intentional for testing, now we can restore the cache